docker-compose.yaml

```diff
@@ -8,13 +8,20 @@ services:
       - 5001:5001
   vitalam-node:
     image: ghcr.io/digicatapult/vitalam-node:v2.5.0
+    command: /bin/sh -c "
```

Comment:
can we move the command to after the environment?

Reply:
removing `environment:` altogether based on the other comments